projects/aca-playwright-shared/src/utils/timeouts.ts

```diff
@@ -34,5 +34,5 @@ export const timeouts = {
   globalTest: 45 * 1000,
   extendedTest: 150 * 1000,
   webServer: 240 * 1000,
-  globalSpec: 60 * 10 * 1000
+  globalSpec: 60 * 20 * 1000
```

Comment:
can you please help to know, why this change required for these tests?

Reply:
I'm getting error in build i.e. timeout waiting 600s for the entire test run as below screenshots, that's why I increase global timeout.

![Screenshot from 2023-12-26 16-04-38](https://github.com/Alfresco/alfresco-content-app/assets/112386818/d7edddc1-4df0-4ed8-9549-b8b4aac8c9e2).

Reply:
can you crate separate test suite for share-action. so each suite can execute under 10 mint. 
as we have action suite test in protractor split for e2e run.

Reply:
yes I can do but should I create test suit in same PR or need to raise a separate PR.

Reply:
you can do it in same.